src/Pim/Bundle/CatalogBundle/Doctrine/Common/Saver/ProductSaver.php

```diff
@@ -115,4 +116,49 @@ protected function validateProduct($product)
             );
         }
     }
+
+    /**
+     * Drops the current completenesses from the database for a list of products.
+     *
+     * @param ProductInterface[] $products
+     */
+    protected function dropCompletenesses(array $products)
```

Comment:
Agree. There is too much things in the saver right now. A `CompletenessRemover` could be nice.